parser.y

```diff
@@ -6436,20 +6437,27 @@ SetExpr:
 EqOrAssignmentEq:
     eq | assignmentEq
 
+VariableName:
+	Identifier
```

Comment:
Don't this branch needs `$$ = $1`?

Reply:
No, the default action for the rule is `$$ = $1`